src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp

```diff
@@ -1520,4 +1523,170 @@ PYBIND11_MODULE(simplnx, mod)
   manualImportFinder.def("clear", &ManualImportFinder::clear);
   manualImportFinder.def("contains_path", &ManualImportFinder::containsPath, "path"_a);
   manualImportFinder.def("contains_module", &ManualImportFinder::containsModule, "mod_name"_a);
+
+  // Geometry Helper Methods
+  py::module_ sub = mod.def_submodule("ArrayHandlingType", "How existing arrays will be handled when creating the new geometry.");
+  sub.attr("CopyArray") = CreateGeometryFilter::k_CopyArray;
+  sub.attr("MoveArray") = CreateGeometryFilter::k_MoveArray;
```

Comment:
Can be replaced by the binding made in https://github.com/BlueQuartzSoftware/simplnx/pull/983 though it needs to be updated to include all values. And the functions after this should be updated as well.

Reply:
@joeykleingers Can you take care of this? If so we could get everything we need for a release.